common/src/main/java/com/microsoft/identity/common/internal/broker/MicrosoftAuthClient.java

```diff
@@ -72,7 +72,7 @@ public MicrosoftAuthServiceFuture connect() throws ClientException {
         mMicrosoftAuthServiceConnection = new MicrosoftAuthServiceConnection(future);
 
         mBound = mContext.bindService(mMicrosoftAuthServiceIntent, mMicrosoftAuthServiceConnection, Context.BIND_AUTO_CREATE);
-        Logger.verbose(TAG + "connect", "The status for MicrosoftAuthService bindService call is: " + Boolean.valueOf(mBound));
+        Logger.info(TAG + "connect", "The status for MicrosoftAuthService bindService call is: " + Boolean.valueOf(mBound));
```

Comment:
I'm thinking if we should keep StorageHelper as info, until after we shipped enable encryption for like a month?

Reply:
As discussed, lets keep at verbose and later turn to info when we turn on encryption